### PR TITLE
Plot profit

### PR DESCRIPTION
**Pull request by kryofly**
Enhance plot_profit and plot_dataframe

### CODE REVIEW

**Comment by glonlas** [comment]
Looks good to me. But I am not an expert on those scripts.

File: scripts/plot_dataframe.py [hunk 1]
```diff
@@ -1,30 +1,23 @@
 #!/usr/bin/env python3
 
 import sys
+import logging
 import argparse
-import matplotlib  # Install PYQT5 manually if you want to test this helper function
-matplotlib.use("Qt5Agg")
```
Comment (vertti) [inline scripts/plot_dataframe.py]: removing this line will stop the script using PyQt5 for rendering

Comment (kryofly) [reply scripts/plot_dataframe.py]: Yes, Tkinter is the defacto python graphical framework.

**Comment by vertti** [comment]
@kryofly if you rebase this, i can merge it in.

**Comment by kryofly** [comment]
Sorry, no keyboard time lately. I'll get to it today.
What about Qt? should I revert it in? What is the feasability of QT and Tkinter on the windoze side (easy to install etc)?  I'm thinking of doing some goofy Tk gui just for the nostalgia :D

**Comment by vertti** [comment]
I'm planning on replacing both plot scripts with HTML based solution (Plotly) very soon. Then we will not need any graphics libraries.

**Pull request merged**